qiskit_optimization/algorithms/recursive_minimum_eigen_optimizer.py

```diff
@@ -203,11 +203,11 @@ def solve(self, problem: QuadraticProgram) -> OptimizationResult:
 
         # run recursive optimization until the resulting problem is small enough
         replacements = {}  # type: Dict[str, Tuple[str, int]]
-        min_eigen_results = []  # type: List[MinimumEigenOptimizationResult]
+        min_eigen_results = []  # type: List[OptimizationResult]
```

Comment:
Since this is now a more general optimizer would the name 'min_eigen_results' be "better" say as a more general name say 'optimization_results' or something - that may be better for any readers of the code in future. I don't think the name is externalized in any result as such - seems just used to form the history tuple later.

Reply:
Yes, updated.